### PR TITLE
New page - custom reference field popup

### PR DESCRIPTION
**Pull request by Rnhatch**
A new Advanced Skuid sample page about custom reference field popups.

### CODE REVIEW

File: Skuid_Techniques/ReferenceSelection/ReferenceSelectionPopup.xml [hunk 1]
```diff
@@ -0,0 +1,240 @@
+<skuid__page unsavedchangeswarning="yes" personalizationmode="server" showsidebar="true" showheader="true">
+	<models>
+		<model id="NewTask" limit="1" query="false" datasource="salesforce" sobject="Task" createrowifnonefound="true">
+			<fields>
+				<field id="RecordTypeId"/>
+				<field id="RecordType.Name"/>
```
Comment (itmightbecody) [inline Skuid_Techniques/ReferenceSelection/ReferenceSelectionPopup.xml:6]: Let's remove these so the page doesn't explode for users without recordtype enabled.
```suggestion
```

File: Skuid_Techniques/ReferenceSelection/ReferenceSelectionPopup.xml [hunk 2]
```diff
@@ -0,0 +1,240 @@
...
+			<conditions>
+				<condition type="datasourceuserinfo" value="" field="OwnerId" fieldtargetobjects="Group,User" operator="=" userinfotype="userid" enclosevalueinquotes="true"/>
+				<condition type="datasourceuserinfo" value="" field="Owner.Name" operator="=" enclosevalueinquotes="true" userinfotype="name"/>
+			</conditions>
+			<actions/>
+		</model>
+		<model id="Accounts" limit="20" query="false" createrowifnonefound="false" datasource="salesforce" sobject="Account" orderby="Name ASC">
+			<fields>
+				<field id="RecordTypeId"/>
+				<field id="RecordType.Name"/>
```
Comment (itmightbecody) [inline Skuid_Techniques/ReferenceSelection/ReferenceSelectionPopup.xml:24]: Remove recordtype again.
```suggestion
```

**Comment by Rnhatch** [comment]
Thanks @itmightbecody .  Your suggestions accepted.  If you can approve - I will merge.

**Comment by Rnhatch** [comment]
@itmightbecody  as discussed in our Slack discussion - I made some changes to the page to correct the errors you were finding.  Check it out now.

**Review by itmightbecody — APPROVED** [review]
Nice! This is good illustration of a reference modal, and great job on the issue fixes. Looks good to me!

**Pull request merged**